### DISCUSSION TITLE
[FYI] I made a burndown chart for TiddlyWiki5

### DISCUSSION THREAD

**Discussion by Jermolene** [TiddlyWiki/TiddlyWiki5#6463] [General]
In this chart the vertical bars represent the open tickets at any one time, with the colour determined by the age of the ticket. It makes a nice visualisation of our occasional progress in reducing the backlog (thanks @pmario!)

<img width="1192" alt="Screenshot 2022-02-15 at 16 32 50" src="https://user-images.githubusercontent.com/174761/154106947-4d1b74a6-71c8-491c-af78-504687c137cd.png">

<img width="1180" alt="Screenshot 2022-02-15 at 16 33 01" src="https://user-images.githubusercontent.com/174761/154106941-fe2c5426-a01c-4e83-bec2-8a3ad82768b1.png">

Using https://observablehq.com/@tmcw/github-burndown

**Comment by Jermolene** [comment]
(Oops this should of course have been a discussion not an issue)

**Reply by stephenteacher** [reply]
Interesting stuff!

**Reply by amreus** [reply]
It looks like the graph is very fine-grained.  Is it showing every day of the year?  The colors flow together and it's hard to follow. I wonder what it might look like if the resolution were reduced to monthly data points, if possible.

**Reply by Jermolene** [reply]
I think the tool I used is indeed showing every day of each year. It's linked above, maybe it can be adapted to work by month.
